### ISSUE TITLE
RadioButton is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6893]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
The radio button doesn't fill properly on selection in Windows High Contrast Mode on Firefox

Firefox
![image](https://user-images.githubusercontent.com/40970507/93944711-0e972a00-fcfb-11ea-92e2-31bbd765b949.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93944718-135bde00-fcfb-11ea-9281-4259afab0e94.png)

**Comment by rjhenriquez** [comment]
@dakahn This seems to have been fixed [here](https://github.com/carbon-design-system/carbon/pull/6892). I'm closing this issue

![Screen Shot 2020-10-13 at 9 19 58 AM](https://user-images.githubusercontent.com/4718579/95873198-49f2aa80-0d35-11eb-9631-2310411d920e.png)
